client/landing/gutenboarding/stores/onboard/index.ts

```diff
@@ -3,6 +3,7 @@
  */
 import { controls } from '@wordpress/data-controls';
 import { plugins, registerStore, use } from '@wordpress/data';
+import { isSupportSession } from 'lib/user/support-user-interop';
```

Comment:
Not sure if this is the type of thing that we want to avoid importing? In our quest for an eventual solo app.

Reply:
Good question! Since it's a `lib/` it might be quite safe to import now like this as long as we make sure it'll eventually get moved to `packages/`. I'm saying might be safe because libs tend to be quite isolated or at least small.

It does import these things:

https://github.com/Automattic/wp-calypso/blob/a1c0a829f3fc9b52820e03b06604d04d2c94ba85/client/lib/user/support-user-interop.js#L1-L15